ffs/manager/manager_test.go

```diff
@@ -23,30 +25,85 @@ func TestMain(m *testing.M) {
 
 func TestNewManager(t *testing.T) {
 	t.Parallel()
-	m, cls := newManager(t, tests.NewTxMapDatastore())
+	client, addr, _ := tests.CreateLocalDevnet(t, 1)
+	m, cls, err := newManager(client, tests.NewTxMapDatastore(), addr, false)
+	require.NoError(t, err)
+	require.NotNil(t, m)
+	defer require.NoError(t, cls())
+}
+
+func TestNewManagerMasterAddrFFSUseMasterAddr(t *testing.T) {
+	t.Parallel()
+	client, addr, _ := tests.CreateLocalDevnet(t, 1)
+	m, cls, err := newManager(client, tests.NewTxMapDatastore(), addr, true)
+	require.NoError(t, err)
+	require.NotNil(t, m)
+	defer require.NoError(t, cls())
+}
+
+func TestNewManagerNoMasterAddrNoFFSUseMasterAddr(t *testing.T) {
```

Comment:
Test to make sure we can create the manager with no master address and not use the master address for all ffs instances